activestorage/test/service/s3_service_test.rb

```diff
@@ -59,20 +59,22 @@ class ActiveStorage::Service::S3ServiceTest < ActiveSupport::TestCase
       end
     end
 
-    test "upload with content type" do
-      key          = SecureRandom.base58(24)
-      data         = "Something else entirely!"
-      content_type = "text/plain"
+    test "upload with content type and content disposition" do
```

Comment:
Also think separate tests for specifying content type and content disposition is a good idea here.